dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -5086,6 +5159,16 @@ definitions:
       verifiedSource:
         type: "string"
         description: "Verified source for the version"
+      doiURL:
```

Comment:
working_directory is snake case, verifiedSource is camel-case, do we have a set standard for non-ga4gh properties?